src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -344,26 +343,17 @@ export async function getQuickFixesForCommand(
 							}
 							case TerminalQuickFixType.Opener: {
 								const fix = quickFix as ITerminalQuickFixOpenerAction;
-								const label = localize('quickFix.opener', 'Open: {0}', fix.uri.toString());
+								const { scheme, authority, path, query, fragment } = fix.uri;
```

Comment:
I wonder if we should put this somewhere general to be used by others?

Reply:
I noticed there's a private `_asFormatted` on `URI` which can be referenced by `URI.toString` but that says explicitly it shouldn't be used for display purposes

Reply:
See my comment about about the label service

Reply:
thanks 😄